README.md

```diff
@@ -57,6 +57,12 @@
 - [Zsh](http://www.zsh.org/) -  A powerful command line shell. [![Open-Source Software][OSS Icon]](http://sourceforge.net/p/zsh/code/ci/master/tree/)
 
 
+### E-Books
+
```

Comment:
Hmmm... Possibly you're right. "E-Books" may evoke the expectation of getting a list of books on OS X, I guess. How about "E-Book Utilities"?